editor/src/clj/editor/fxui.clj

```diff
@@ -65,6 +66,22 @@
       (retract! [_ instance _ _]
         (set-all! instance [])))))
 
+(defn tuple-lifecycle [& lifecycles]
+  (let [lifecycles (vec lifecycles)
+        len (count lifecycles)
+        instance-meta {`fx.component/instance #(mapv fx.component/instance %)}]
```

Comment:
We could move this out and reuse it for all calls to `tuple-lifecycle`, perhaps?

Reply:
Yes, good point!